src/v2/Components/Search/SearchBar.tsx

```diff
@@ -266,11 +266,14 @@ export class SearchBar extends Component<Props, State> {
   )
   onSuggestionSelected({
     suggestion: {
-      node: { href, displayType },
+      node: { href, __typename },
     },
+    method,
   }) {
+    if (method === "click") return
```

Comment:
Navigating on click will prevent the quick navigation items from working properly.